package.json

```diff
@@ -82,12 +79,25 @@
     "mode:account": "yarn mode:cloud && yarn env:account:enable",
     "security:audit": "node scripts/audit.js",
     "postinstall": "husky install",
-    "install:pro": "bash scripts/pro/install.sh",
-    "dep:clean": "yarn clean && yarn bootstrap"
+    "dep:clean": "yarn clean && yarn bootstrap",
```

Comment:
Maybe could add the `-y` option to clean to automatically handle the question prompt - if you're running `dep:clean` its likely intentional